### ISSUE TITLE
Failed with RP v5

### ISSUE THREAD

**Issue by sunshine69** [osi-open-source/reporting-portal-agent-codeception#1]
Hi,

I tried this repo and while all composer update, install pass when I run test I got the errors:

```
vendor/bin/codecept run access
Codeception PHP Testing Framework v4.1.20
Powered by PHPUnit 9.5.4-38-g3d0f17b1f by Sebastian Bergmann and contributors.

GO1.access Tests (2) ----------------------------------------------------------------------------------------------------------------------
Cannot connect to reporting portal. Exception message: Undefined property: stdClass::$id at vendor/reportportal/basic/src/service/ReportPortalHTTPService.php:524
✔ ResetPasswordCest: Test Send a POST /access/v1.0/password/reset request, then 204 status code is returned (0.22s)
✔ ResetPasswordCest: Test Send a POST /access/v1.0/password/reset request, then 400 status code is returned (0.21s)
-------------------------------------------------------------------------------------------------------------------------------------------


Time: 00:02.778, Memory: 16.00 MB

OK (2 tests, 3 assertions)
```

Not sure if it is because RP version 5 changed so much in a breaking way and all tools stop working with this? If so then is there any plan to support RP v5?

Thanks.

**Comment by sunshine69** [comment]
It looks like RP v4 has the same error

```
Cannot connect to reporting portal. Exception message: Undefined property: stdClass::$id at vendor/reportportal/basic/src/service/ReportPortalHTTPService.php:524
```

**Comment by sunshine69** [comment]
False positive. It requires the exact project name in the config. But when it failed it does not give any meaningful error message though.

**Comment by osibg** [comment]
Yes, we need better error handling. Thank you for your notes and sorry about the misleading. There is a config `allowFailure` that is `true` by default to allow running tests even of something is wrong with Reporting Portal. But still, there is not enough details about the error. 
Even in the [original Client](https://github.com/Mikalai-Kabzar/client-php/blob/master/src/service/ReportPortalHTTPService.php) response from the portal is not collected and the exact error is hard to follow. 

This package is not actively supported and we cannot say if there will be any update on that. But if you make a Pull Request we can help on publishing and testing it.
